### ISSUE TITLE
Getting the invocation index of current parameterized test run

### ISSUE THREAD

**Issue by serpro69** [junit-team/junit5#1982]
Trying to get the index of the current invocation count of a ParameterizedTest, the same value that can be used when declaring a custom name, i.e. `@ParameterizedTest(name = "{index}")` but through a parameter.

TestInfo does not provide that information, only the method name, and I would rather not try to extract this value from the method name string.

Is there any other way this index can be retrieved?

I need this to have a certain piece of test method execute only once. For example creating the data for the test.
I thought about moving this logic out to the method source, but I would rather avoid doing that if I can because otherwise the test method will be cluttered with the parameters that I will have to pass from the method source.

**Comment by serpro69** [comment]
I think this is a duplicate of https://github.com/junit-team/junit5/issues/1668. Closing.
